src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
+  padding: 50px 40px;
+  margin: 0 auto;
+
+  &__card {
+    box-sizing: border-box;
```

Comment:
instead of setting it for each element you need, better to apply it to all elements (`*` selector)